managed/CounterStrikeSharp.API/Utilities.cs

```diff
@@ -103,5 +103,36 @@ public static unsafe string ReadStringUtf8(IntPtr ptr)
                 return Encoding.UTF8.GetString(buffer);
             }
         }
+        
+        public static IEnumerable<CCSPlayerController> FindTarget(CCSPlayerController? player, string pattern) {
+            var emptyResult = Enumerable.Empty<CCSPlayerController>();
+            var playerEntities = Utilities.FindAllEntitiesByDesignerName<CCSPlayerController>("cs_player_controller");
+            
+            if (pattern.StartsWith("#")) 
+            {
+                if (!int.TryParse(pattern.Substring(1), out var userId)) 
+                    return emptyResult;
+                
+                var playerController = playerEntities.Select(player => player).FirstOrDefault(player => player.UserId == userId);
+                
+                return playerController != null ? new List<CCSPlayerController>() { playerController } : emptyResult;
+            }
+            else if (pattern.StartsWith("@")) 
```

Comment:
We can remove this redundant else because of the return in the previous if statement.